spec/java/util/Random.lsl

```diff
@@ -0,0 +1,228 @@
+///#! pragma: non-synthesizable
+libsl "1.1.0";
+
+library std
+    version "11"
+    language "Java"
+    url "-";
+
+// imports
+
+import java.common;
+import java/lang/_interfaces;
+import java/util/stream/_interfaces;
+
+
+// local semantic types
+
+@implements("java.io.Serializable")
```

Comment:
it implements "java.util.random.RandomGenerator" too

Reply:
No, it's [not](https://github.com/openjdk/jdk11/blob/master/src/java.base/share/classes/java/util/Random.java#L77).

Reply:
Yes, you are right; 
I looked at [jdk19](https://github.com/openjdk/jdk19/blob/master/src/java.base/share/classes/java/util/Random.java#L77)